pkg/catalogsourceconfig/configuring.go

```diff
@@ -5,11 +5,11 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/operator-framework/operator-marketplace/pkg/operatorsource"
-
 	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	interface_client "github.com/operator-framework/operator-marketplace/pkg/client"
```

Comment:
I agree with fixing this is a follow up PR. But won't it be a bit more confusing to have only this part using `interface`?

Reply:
I suggested fixing up **other** places in a follow up PR. There are way too many being introduced in this PR to keep holding off.

>But won't it be a bit more confusing to have only this part use interface?

See the usage `interface.Client` vs `interface_client.Client`.

Reply:
That's fair - there is an overwhelming majority in this PR and the confusion isn't really an issue. Will fix

Reply:
Please make these changes in the commit where you are introducing the change. Not in a separate or in the final commit.

Reply:
I can't use `interface` because it's a key word in go. What do you think of `wrapper.Client` or `fake.Client`. @kevinrizza @awgreene @anik120 please let me know your thoughts

Reply:
Neither `interface` nor `wrapper` or `fake` seem to capture what the interface is for. Wasn't the decision made to use `interfaceClient` instead?

Reply:
that was suggested but not decided. `interfaceClient.Client` is a bit of a long and redundant name IMO

Reply:
@anik120 `intefaceClient` was not decided upon. I don't see why `wrapper` doesn't describe what it is. Here is the doc:
```
// Client is a wrapper around the raw kube client provided
// by operator-sdk. Using the wrapper facilitates mocking of client
// interactions with the cluster, while using fakeclient during unit testing.
```
as written by you :-)

Reply:
Maybe `interfaceClient` was more natural to me since I thought of it that way all the while I was designing it. But if you guys think `wrapper` is natural too then I'm stepping off of this :p

Reply:
... if we really want to be pedantic ( :) ) I think that the real problem is the word ordering. It's not a client, it's an interface or a wrapper for a type. So it seems to me that the right thing would be something like `clientWrapper`.